studio/sqs_queue.py

```diff
@@ -13,9 +13,9 @@
 class SQSQueue(object):
 
     def __init__(self, name, verbose=10, receive_timeout=300,
-                 retry_time=10):
+                 retry_time=10, region_name='us-west-2'):
```

Comment:
would this not automatically populate from AWS_DEFAULT_REGION env variable or ~/.aws/config.json?

Reply:
I found the code that was using this parameter inside one of my applications for the python runner.  I do use however I think maybe the default should be removed.  I will make that change.